locale/en/download/package-manager.md

```diff
@@ -294,3 +295,11 @@ Using **[Scoop](https://scoop.sh/)**:
 ```bash
 scoop install nodejs
 ```
+
+## z/OS
+
+To install Node.js on z/OS,you can follow the below steps.
+There are two available methods to install Node.js,SMP/E and PAX.
```

Comment:
```suggestion
IBM® SDK for Node.js - z/OS® is available in two installation formats, SMP/E and PAX. Select the installation format that applies to you:
```
(that's from https://www.ibm.com/support/knowledgecenter/SSTRRS_14.0.0/com.ibm.nodejs.zos.v14.doc/install.htm)